adminapi/request.py

```diff
@@ -128,7 +128,9 @@ def json_encode_extra(obj):
     if isinstance(obj, BaseFilter):
         return obj.serialize()
     if isinstance(obj, datetime):
-        return obj.strftime('%Y-%m-%d %H:%M:%S')
+        if obj.tzinfo is None:
+            obj = obj.replace(tzinfo=timezone.utc)
```

Comment:
Check, this is convenient.

Reply:
Although this would break Python 2 compatibility :(

Reply:
Fixed py2 compatibility. I understand why they didn't want to mess with keeping timzones up-to-date but not shipping an UTC timezone pre python3.2 is just dumb.

Reply:
Eh, I found more problems. I'll check this later..

Reply:
Soo.. Running the following code breaks in py2:
```python
dt = datetime.utcnow().replace(tzinfo=utc)
dstr = dt.strftime('%Y-%m-%d %H:%M:%S%z')
datetime.strptime(dstr, '%Y-%m-%d %H:%M:%S%z')
```

The strftime actually works but the strptime doesn't know %z 🙄

Reply:
Maybe this is the time to finally drop python2 support. Riding a dead horse.